frontend/device/android/device.lua

```diff
@@ -254,12 +250,19 @@ function Device:performHapticFeedback(type)
 end
 
 function Device:retrieveNetworkInfo()
-    local ssid, ip, gw = android.getNetworkInfo()
-    if ip == "0" or gw == "0" then
+    local ok, type = android.getNetworkInfo()
+    if ok == "0" then
         return _("Not connected")
     else
-        local BD = require("ui/bidi")
-        return T(_("Connected to %1\n IP address: %2\n gateway: %3"), BD.wrap(ssid), BD.ltr(ip), BD.ltr(gw))
+        local status = nil
+        if type == "1" then
+            status = _("connected to Wi-Fi")
+        elseif type == "2" then
+            status = _("connected to mobile data")
+        elseif type == "3" then
+            status = _("connected to ethernet")
+        end
+        return status or _("connecting")
```

Comment:
No point adding another string to translate.
```suggestion
        return status or _("Connecting…")
```